wye/workshops/urls.py

```diff
@@ -10,4 +11,6 @@
         WorkshopUpdate.as_view(), name="workshop_update"),
     url(r'^(?P<action>[active,deactive]+)/(?P<pk>\d+)/$',
         WorkshopToggleActive.as_view(), name="workshop_toggle"),
+    url(r'^presenter/(?P<action>[opt\-in,opt\-out]+)/(?P<pk>\d+)/$',
```

Comment:
Can url be /workshop/<id>/action so we can extend same url for admin to assign to any presenters and presenter assigning himself?